src/commands/start-project.ts

```diff
@@ -177,12 +181,26 @@ const startProject = async (toolbox: RnBootstrapToolbox) => {
     }
 
     if (gluestackOptions.includes(selectedOptions.styleLibrary)) {
+      // Replace storybook web files with preconfigured ones specific for gluestack
       replaceFile('preview.tsx', '.storybook/preview.ts');
       filesystem.rename(
         `${process.cwd()}/.storybook/preview.ts`,
         'preview.tsx'
       );
+      // Remove files with specified extensions from the 'src/stories' folder
+      const extensionsToRemove = ['ts', 'tsx', 'css'];
+      const files = filesystem.list('src/stories');
+      files
```

Comment:
@GerganaNIvanova let's define this constant as stories and filter them as storyFiles.

Reply:
Done!